process-engine/bpmn-studio
# ♻️ Refactor Date Service

## Changes

1. Refactor the date service
2. Use the date service in the process list

## Issues

Closes #1561 

PR: #1562

## How to test the changes

- Go to the dashboard
- **Notice that the 'Started At' date is still displayed correctly.**
